docs/_developers/Peer-to-Peer.md

```diff
@@ -1,161 +0,0 @@
----
-title: Peer-to-peer
```

Comment:
@fjl Was this page depreciated or moved elsewhere? I can't seem to find a replacement (after looking very briefly)

Reply:
I didn't move this one. Checking the content again now, it's actually one of the few legacy pages where the code still works :). Maybe we should bring it back after all.